CHANGELOG.md

```diff
@@ -16,16 +16,20 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 - Config parameter `external_address` is now a required value. (#826)
 
-- Config parameter `round_timeout` is removed.
-  `first_round_timeout` and `round_timeout_increase` should be used instead. (#848)
+- Config parameter `round_timeout` has been renamed to `first_round_timeout`.
+  Now timeout for round r is
+  `first_round_timeout + (r-1)*round_timeout_increase`
+   where `round_timeout_increase` is determined as a some percentage of
+  `first_round_timeout`. Value of this percentage is defined in constant inside
+  `ConsensusConfig` (#848)
```

Comment:
Some last-minute nit: I'd reword this changelog entry to explicitly reference the `TIMEOUT_LINEAR_INCREASE_PERCENT` constant instead of 'some percentage' (it is public after all). We can also note its current value (10%) so that developers reading the changelog have an idea of its magnitude.